kubernetes/ansible/roles/deploy-player/tasks/main.yml

```diff
@@ -32,9 +32,40 @@
 - name: create configmap
   shell: "kubectl create configmap player-config --from-env-file={{role_path}}/templates/{{ release_name }}.env -n {{namespace}} --dry-run -o=yaml | kubectl apply -f -"
 
+- name: Load role to decrypt private keys, copy to helm chart and encrypt private keys
+  include_role:
+    name: mount-keys
+    tasks_from: "{{outer_item.1}}"
+  vars:
+    private_key_path: "{{ outer_item.0.values_to_pass.basepath }}"
+    private_key_prefix: "{{ outer_item.0.values_to_pass.keyprefix }}"
+    private_key_sign_start: "{{ outer_item.0.values_to_pass.keystart }}"
+    private_key_sign_end: "{{ outer_item.0.values_to_pass.keycount if outer_item.0.values_to_pass.keycount > '0' else '0'  }}"
+  when: release_name == "player"
```

Comment:
is it really required @keshavprasadms as this role is specific to player right?

Reply:
Yes @SMYALTAMASH  In order to inject the keys into the pod, we need these roles to be invoked.

Reply:
ok @keshavprasadms